src/config/dnsmasq_config.c

```diff
@@ -473,7 +473,7 @@ bool __attribute__((const)) write_dnsmasq_config(struct config *conf, bool test_
 
 		if(active == NULL || cidr == NULL || target == NULL || domain == NULL)
 		{
-			log_err("Invalid reverse server string: %s", revServer->valuestring);
+			log_err("Skipped invalid dns.revServers[%u]: %s", i, revServer->valuestring);
```

Comment:
Should this be an error or just a warning?

```suggestion
			log_warn("Skipped invalid dns.revServers[%u]: %s", i, revServer->valuestring);
```

Reply:
This is subtle. I think this is an error even when it is not a fatal one. It's still invalid configuration